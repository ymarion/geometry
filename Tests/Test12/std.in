# Initializing some objects
C c1 0 0 10
C c2 0 0 20
# Undoing
UNDO
# Redoing
REDO
# Should have c1 and c2
LIST
#
EXIT
