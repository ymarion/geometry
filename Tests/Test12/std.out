C c1 0 0 10
C c2 0 0 20
